Config file config/config.yaml is extended by additional config specified via the command line.
Building DAG of jobs...
Provided cores: 8
Rules claiming more threads will be scaled down.
Conda environments: ignored
Job stats:
job                     count
--------------------  -------
add_generators              1
build_load_weights          1
cluster_network             1
prepare_bmu_data            1
prepare_live_network        1
retrieve_wiki_data          1
simplify_network            1
solve_network               1
total                       8

Select jobs to execute...

[Fri Mar 29 11:21:51 2024]
rule retrieve_wiki_data:
    output: resources/wiki_data.csv
    log: logs/pypsa-lmp/retrieve_wiki_data.log
    jobid: 6
    reason: Missing output files: resources/wiki_data.csv
    resources: tmpdir=C:\Users\alyden\AppData\Local\Temp, mem_mb=1000, mem_mib=954

[Fri Mar 29 11:21:51 2024]
rule build_load_weights:
    input: resources/regions_onshore.geojson, data/gsp_geometries.geojson, data/gsp_gnode_directconnect_region_lookup.csv, data/FES-2021--Leading_the_Way--demandpk-all--gridsupplypoints.csv, data/FES-2021--Falling_Short--demandpk-all--gridsupplypoints.csv
    output: resources/load_weights.csv
    log: logs/pypsa-lmp/build_load_weights.log
    jobid: 8
    reason: Missing output files: resources/load_weights.csv
    resources: tmpdir=C:\Users\alyden\AppData\Local\Temp, mem_mb=1000, mem_mib=954

[Fri Mar 29 11:21:52 2024]
Error in rule retrieve_wiki_data:
    jobid: 6
    output: resources/wiki_data.csv
    log: logs/pypsa-lmp/retrieve_wiki_data.log (check log file(s) for error details)
    conda-env: C:\Users\alyden\OneDrive - University of Edinburgh\Python\LMP\.snakemake\conda\cf0dbe1799d06fa5b5ff98f73b111298_

RuleException:
CalledProcessError in file C:\Users\alyden\OneDrive - University of Edinburgh\Python\LMP\Snakefile, line 190:
Command 'C:/Users/alyden/AppData/Local/miniforge3/envs/pypsa-lmp/python.exe "C:\Users\alyden\OneDrive - University of Edinburgh\Python\LMP\.snakemake\scripts\tmpp01k695t.retrieve_wiki_data.py"' returned non-zero exit status 1.
  File "C:\Users\alyden\OneDrive - University of Edinburgh\Python\LMP\Snakefile", line 190, in __rule_retrieve_wiki_data
  File "C:\Users\alyden\AppData\Local\miniforge3\envs\pypsa-lmp\Lib\concurrent\futures\thread.py", line 58, in run
[Fri Mar 29 11:22:21 2024]
Finished job 8.
1 of 8 steps (12%) done
Shutting down, this might take some time.
Exiting because a job execution failed. Look above for error message
Complete log: .snakemake\log\2024-03-29T112150.864591.snakemake.log
